src/lib/db/feature-environment-store.ts

```diff
@@ -366,8 +366,11 @@ export class FeatureEnvironmentStore implements IFeatureEnvironmentStore {
         projects: string[],
     ): Promise<void> {
         await this.db.raw(
-            `INSERT INTO ${T.featureEnvs} (
-                SELECT distinct ? AS environment, feature_name, enabled FROM ${T.featureEnvs} INNER JOIN ${T.features} ON ${T.featureEnvs}.feature_name = ${T.features}.name WHERE environment = ? AND project = ANY(?))`,
+            `INSERT INTO ${T.featureEnvs} (environment, feature_name, enabled, variants)
+             SELECT DISTINCT ? AS environemnt, fe.feature_name, fe.enabled, fe.variants
+             FROM ${T.featureEnvs} AS fe
+                      INNER JOIN ${T.features} AS f ON fe.feature_name = f.name
+             WHERE fe.environment = ? AND f.project = ANY(?)`,
```

Comment:
2 changes here: Add the missing variants column and use table aliases to avoid column ambiguity